pandas/tests/io/test_pytables.py

```diff
@@ -5167,3 +5168,71 @@ def test_dst_transitions(self):
                 store.append('df', df)
                 result = store.select('df')
                 assert_frame_equal(result, df)
+
+
+@pytest.fixture(scope='module')
+def pytables_hdf5_file():
+    """Use PyTables to create a simple HDF5 file."""
+
+    table_schema = {
+        'c0': tables.Time64Col(pos=0),
+        'c1': tables.StringCol(5, pos=1),
+        'c2': tables.Int64Col(pos=2),
+    }
+
+    t0 = time.time()
+
+    testsamples = [
+        {'c0': t0, 'c1': 'aaaaa', 'c2': 1},
+        {'c0': t0 + 1, 'c1': 'bbbbb', 'c2': 2},
+        {'c0': t0 + 2, 'c1': 'ccccc', 'c2': 10**5},
+        {'c0': t0 + 3, 'c1': 'ddddd', 'c2': 4294967295},
+    ]
+
+    # This returns a path and does not open the file.
+    tmpfilepath = create_tempfile('pytables_hdf5_file')
+    objectname = 'pandas_test_timeseries'
+
+    with tables.open_file(tmpfilepath, mode='w') as hf:
+        t = hf.create_table('/', name=objectname, description=table_schema)
+        for sample in testsamples:
+            for key, value in sample.items():
+                t.row[key] = value
+            t.row.append()
+
+    return tmpfilepath, objectname, pd.DataFrame(testsamples)
+
+
+class TestReadPyTablesHDF5(Base):
```

Comment:
do you *need* to inherit from Base?

Reply:
> do you need to inherit from Base?

I do not really know. Not sure if `tm.reset_testing_mode()`  is important at all in my context here:
```\class Base:

    @classmethod
    def setup_class(cls):

        # Pytables 3.0.0 deprecates lots of things
        tm.reset_testing_mode()

    @classmethod
    def teardown_class(cls):

        # Pytables 3.0.0 deprecates lots of things
        tm.set_testing_mode()
```

Given that you doubt that I need it I have added a commit and don't inherit from `Base` anymore. The tests still pass on my system.